ipalib/x509.py

```diff
@@ -535,6 +568,22 @@ def write_certificate_list(certs, filename):
         raise errors.FileError(reason=str(e))
 
 
+def write_pem_private_key(priv_key, filename):
+    """
+    Write a private key to a file in PEM format.
+
+    :param priv_key: cryptography ``PrivateKey`` object
+    """
+    try:
+        with open(filename, 'wb') as fp:
+            fp.write(priv_key.private_bytes(
```

Comment:
This is potentially dangerous and would write the file as 0x644 if new. I'll be forcing 0x600 in the upcoming update.